index.d.ts

```diff
@@ -2976,7 +2976,7 @@ declare module sequelize {
     [string | col | fn | literal, string] |
     [Model<any, any> | { model: Model<any, any>, as: string }, string, string] |
     [Model<any, any>, Model<any, any>, string, string];
-  export type Order = OrderItem | OrderItem[];
+  export type Order = string | fn | col | literal | OrderItem[];
```

Comment:
Sorry I misunderstood that the first time. There's [one issue that breaks build on travis ci](https://travis-ci.org/louy/typed-sequelize/builds/138466805#L180-L182). If you can fix that I'm happy to merge this.